src/components/ContentPreview/ContentPreview.js

```diff
@@ -538,7 +552,7 @@ class ContentPreview extends PureComponent<Props, State> {
             };
         }
 
-        onMetric(metrics);
+        onMetric(METRIC_TYPE_PREVIEW_METRIC, metrics);
```

Comment:
Is this considered a breaking change, as the function params are changing? Also, it looks like we include `event_name: PREVIEW_LOAD_METRIC_EVENT` as a property of `metrics`. Should we use `event_name` for the new metrics, as well?